project/MetalsPlugin.scala

```diff
@@ -1,85 +1,112 @@
 import sbt._
 import sbt.Keys._
 import java.io._
+import sbt.Def
 
 object MetalsPlugin extends AutoPlugin {
   override def trigger = allRequirements
   override def requires = sbt.plugins.JvmPlugin
-  val metalsCompilerConfig =
-    taskKey[String]("Configuration parameters for autocompletion.")
-  val metalsSetup =
-    taskKey[Unit](
-      "Generate build metadata for completions and indexing dependency sources"
+  object autoImport {
+
+    val metalsCompilerConfig =
+      taskKey[String](
+        "String containing build metadata in properties file format."
+      )
+    val metalsWriteCompilerConfig =
+      taskKey[Unit](
+        "Generate build metadata for completions and indexing dependency sources"
+      )
+
+    lazy val semanticdbSettings = List(
+      addCompilerPlugin(
+        "org.scalameta" % "semanticdb-scalac" % SemanticdbEnable.semanticdbVersion cross CrossVersion.full
+      ),
+      scalacOptions += "-Yrangepos"
+    )
```

Comment:
This conflicts with `semanticdbSettings` defined in `build.sbt` (just remove it there)